docs/docs/integrations/language-models/mistral-ai.md

```diff
@@ -374,6 +374,17 @@ Toggling the safe prompt will prepend your messages with the following `@SystemM
 Always assist with care, respect, and truth. Respond with utmost utility yet securely. Avoid harmful, unethical, prejudiced, or negative content. Ensure replies promote fairness and positivity.
 ```
 
+## Creating `MistralAiModerationModel`
+
+### Plain Java
+```java
+ModerationModel model = MistralAiModerationModel.Builder()
+    .apiKey(System.getenv("MISTRAL_AI_API_KEY"))
+    ..
```

Comment:
please specify model name explicitly

Reply:
Done !

Reply:
where? please add it to hte documentation (here)

Reply:
Sorry for confiusion, I meant that it would be great to specify the model name in the example:
```
MistralAiModerationModel.Builder()
    .apiKey(System.getenv("MISTRAL_AI_API_KEY"))
    .modelName(...)
...
```

Reply:
done